collector/eventhub/src/main/java/zipkin/collector/eventhub/EventHubCollector.java

```diff
@@ -33,15 +28,18 @@
 public class EventHubCollector implements CollectorComponent {
```

Comment:
before we release, we'll want to mark this class final (probably add at least cursory javadoc) and mark the constructor package private (by removing the public qualifier from it)